react/components/CreateOrganizationButton/CreateOrganizationButton.tsx

```diff
@@ -57,13 +59,14 @@ const CreateOrganizationButton = () => {
           icon={<IconPencil />}
           onClick={() => setOpen(true)}
         />
-        {!!data && (
+        <ImportInBulkTooltip visible={!!data}>
```

Comment:
Shouldn't the tooltip be visible when there's no data? It seems like this would make it show up when the data is available instead.

Reply:
Oh, that's truth, thank you!